src/plone/restapi/tests/test_types.py

```diff
@@ -109,7 +109,7 @@ def test_get_jsonschema_for_fti(self):
             request,
             excluded_fields=['title']
         )
-        self.assertNotIn('title', jsonschema['properties'].keys())
+        self.assertNotIn('title', list(jsonschema['properties'].keys()))
```

Comment:
this should be written: `self.assertNotIn('title', list(jsonschema['properties']))`